tests/src/Functional/ContentRenderTestBase.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel\Functional;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Base class for testing content types.
+ */
+abstract class ContentRenderTestBase extends BrowserTestBase {
```

Comment:
This is a good idea, but let's name it WhitelabelBrowserTestBase.
The only thing that it should do is the common setup for browser tests in oe_whitelabel.